tcms/testplans/forms.py

```diff
@@ -92,11 +92,6 @@ def clean_tag(self):
 
 
 class EditPlanForm(NewPlanForm):
-    product_version = forms.ModelChoiceField(
-        label="Product Version",
-        queryset=Version.objects.all(),
-        empty_label=None,
-    )
```

Comment:
Why is this obsolete? Commit log should clearly state why we want to remove this value.